docs/mkdocs/docs/technical/releasing.md

```diff
@@ -11,23 +11,29 @@ Click `Run Workflow` on the right hand side:
 1. Type in the new version number
 2. Click `Run workflow`.
 
-Leave `Bump branch to the next version` as `No`. 
-This will create a branch off of `master` incrementing the version in `setup.cfg` but, 
+Leave `Bump branch to the next version` as `No`.
+This will create a branch off of `master` incrementing the version in `setup.cfg` but,
 as of the time of writing, we are leaving that unchanged.
 
 ## 2. Update the version number in the tag
 
 **This should not be a manual process - but is required for now!**
 
-We need to update the version for 
-[`arcticdb.__version__`](https://github.com/man-group/ArcticDB/blob/master/python/arcticdb/\_\_init\_\_.py#LL14C1-L14C1).
+We need to update the public version identifier:
+ - [CMake's `PROJECT_VERSION`](https://github.com/man-group/ArcticDB/blob/master/cpp/CMakeLists.txt#L5)
+ - [`arcticdb.__version__`](https://github.com/man-group/ArcticDB/blob/master/python/arcticdb/\_\_init\_\_.py#LL14C1-L14C1).
+ - `version` in the `metadata` section of [`setup.cfg`](https://github.com/man-group/ArcticDB/blob/master/setup.cfg)
```

Comment:
I don't think this is necessary, can't we just leave it stale? It doesn't affect the released artifacts.

Reply:
It matters if people are redistributing it on other platforms than conda-forge and PyPI.

Reply:
Also, it matters when one installs ArcticDB in editable mode. For instance, installing it in editable mode currently produces a wrong versioning:

```
mamba list
```
```                                                                                                                                                                                                                                                                                                           
# packages in environment at /home/jjerphan/.local/share/mambaforge/envs/arcticdb:                                                                                                                                                                                                                                            
#                                                                                                                                                                                                                                                                                                                             
# Name                    Version                   Build  Channel                                                                                                                                                                                                                                                      
arcticdb                  1.0.1                    pypi_0    pypi          
```

Reply:
That version can be automatically maintained by the "tag" job; however, it seems @mehertz have not been ticking that box. Maybe we should make it automatic if tagging from `master`.

Reply:
We don't release off of master though - the branches that we release off of do have the `setup.cfg` version number changed. 

Surely that's the only record that matters?

Reply:
I'm going to remove this section - the only records we want to change are the latter two which are now both handled by the tag job